src/data/faq_de.json

```diff
@@ -2210,47 +2294,214 @@
+                    "anchor": "short_lived_random_bluetooth_id",
+                    "description": "Mehrmals pro Stunde erzeugter eindeutiger Schlüssel, der aus dem zufälligen Geräteschlüssel abgeleitet und über Bluetooth zwischen benachbarten Smartphones ausgetauscht wird."
+                }
+            ],
+            "M": [
+                {
+                    "term": "Molekularbiologischer Test",
+                    "anchor": "molecular_biological_test",
+                    "description": "Siehe <a href='#glossary_pcr_test'>PCR-Test</a>"
+                }
+            ],
+            "O": [
+                {
+                    "term": "Open-Source",
+                    "anchor": "open_source",
+                    "description": "Code, der der Öffentlichkeit sichtbar, zugänglich, veränderbar und verteilbar ist."
                 }
             ],
             "P": [
+                {
+                    "term": "PCR-Test",
+                    "anchor": "pcr_test",
+                    "description": "PCR-Tests gelten als die aussagekräftigsten Corona-Tests. Bei dieser Art des Tests entnimmt medizinisches Personal die Probe und lässt sie durch ein Labor auswerten."
+                },
                 {
                     "term": "Prüf-App / Prüfen eines Zertifikats / Verifizieren eines Zertifikats",
                     "anchor": "verify",
-                    "description": "Die reine Existenz eines (digitalen) Dokuments hat keine Nachweis-/Beweiskraft. Vielmehr <b>muss</b> jedes Dokument/Zertifikat auch auf die Gültigkeit und Validität überprüft werden. <br/><br/> Diese Prüfung erledigt eine spezielle App (in Deutschland die CovPassCheck-App), die Prüf-App. <br/><br/> Die einzelnen Länder haben möglicherweise unterschiedliche Implementierungen der Prüf-App. Aber alle Prüf-Apps nutzen dieselben – auf einem europäischen Gateway konsolidieren – Schlüssel der teilnehmenden Länder sowie die hinterlegten <a href='#glossary_business_rules'>Business Rules</a>. Daher können alle Prüf-Apps künftig auch alle EU-Zertifikate validieren, sofern sie der Spezifikation des europäischen eHealth Network folgen."
+                    "description": "Die reine Existenz eines (digitalen) Dokuments hat keine Nachweis-/Beweiskraft. Vielmehr <b>muss</b> jedes Dokument/Zertifikat auch auf die <a href='#glossary_validity'>Gültigkeit und Validität</a> überprüft werden. <br/><br/> Diese Prüfung erledigt eine spezielle App (in Deutschland die <a href='#glossary_covpasscheck'>CovPassCheck-App</a>), die Prüf-App. <br/><br/> Die einzelnen Länder haben möglicherweise unterschiedliche Implementierungen der Prüf-App. Aber alle Prüf-Apps nutzen dieselben – auf einem europäischen Gateway konsolidieren – Schlüssel der teilnehmenden Länder sowie die hinterlegten <a href='#glossary_business_rules'>Business Rules</a>. Daher können alle Prüf-Apps künftig auch alle EU-Zertifikate validieren, sofern sie der Spezifikation des europäischen eHealth Network folgen."

```

Comment:
konsolidierten

Reply:
Will be changed in the next commit, thanks!